CorsixTH/Lua/dialogs/edit_room.lua

```diff
@@ -427,6 +434,13 @@ function UIEditRoom:finishRoom()
         end
         map:setCell(x, y, layer, flag + tiles.west)
       end
+	elseif dir == "west_window_long" then
+      if x ~= rect.x then
+		map:setCell(x, y, layer, flag + tiles.west)
+        if map:getCell(x, y + 1, layer) ~= 0 then
+          map:setCell(x, y + 1, layer, flag + tiles.west)
+        end
+      end
```

Comment:
```suggestion
    if x ~= rect.x then
		  map:setCell(x, y, layer, flag + tiles.west)
      if map:getCell(x, y + 1, layer) ~= 0 then
        map:setCell(x, y + 1, layer, flag + tiles.west)
      end
    end
```

Reply:
Conversation view made this suggestion look even worse, but it looks fine on the files tab